code/datums/components/tackle.dm

```diff
@@ -533,6 +541,68 @@
 
 	playsound(owner, 'sound/weapons/smash.ogg', 70, TRUE)
 	tackle.finalize(hit=TRUE)
-	resetTackle()
+	finishTackle()
 
 #undef MAX_TABLE_MESSES
+
+/**
+ * If you want to make someone launch just one tackle without their input, give them this component with the target var set to whatever you want to launch them at.
+ *
+ * If said person already has a normal tackler component this won't affect them, but that's benign enough that it shouldn't cause problems in gameplay.
+ * Also comes with a delay var, in case you want to make the person tackle after a second or something like that.
+ */
+/datum/component/tackler/one_shot
```

Comment:
Definitely do not make a subtype of a component--this is an extremely bad practice.

I'm not totally sure what the difference between this one is, but if you let me know over Discord I can help you out on how to, most likely, provide the right configuration options to leave this to one component.